site/docs/v1/tech/themes/kickstart-custom-theme.adoc

```diff
@@ -0,0 +1,400 @@
+---
+layout: doc
+title: Kickstart instance with custom theme
+description: Using Kickstart to Create a FusionAuth Instance with a Custom Theme
+navcategory: customization
+---
+:page-liquid:
+
+== Overview
+
+https://fusionauth.io/docs/v1/tech/installation-guide/kickstart[FusionAuth Kickstart™] enables you to spin up a reproducible FusionAuth environment using a template file. This guide will walk you through the process of using Kickstart to set up a FusionAuth instance with a custom theme.
+
+We'll apply a dark theme in place of the default application theme presented in the https://fusionauth.io/docs/v1/tech/getting-started/5-minute-docker[5-minute Docker installation guide]. We recommend following the 5-minute Docker installation guide before using this guide to automate the set-up process and apply a custom theme using Kickstart.
+
+== Prerequisites
+
+You'll need to have Node.js and Docker installed to follow this guide and the 5-minute guide. More information can be found https://fusionauth.io/docs/v1/tech/getting-started/5-minute-docker#requirements[here].
+
+== Getting Started
+
+Once you've completed the steps in the 5-minute guide, we can take a look at the state of the FusionAuth instance you created.
+
+Navigate to "Applications" and edit the application you created. Go to the "OAuth" tab to examine the settings that you configured.
+
+image::themes/kickstart-theme/app-settings.png[Application settings.,width=1200]
+
+In the 5-minute guide, the "Authorized redirect URLs", "Logout URL", "Enabled grants", and "Require registration" fields were manually entered. We will use Kickstart to automate all of these settings. We will also apply predefined "Client Id" and "Client secret" values to make launching the application easier.
+
+Now, navigate to "Users" -> "Manage" for the user that you set up in the 5-minute guide.
+
+image::themes/kickstart-theme/manageuser.png[Manage user.,width=1200]
+
+
+In the 5-minute guide, you created both the user and the FusionAuth registration by manually filling out the registration form. You then added a registration for the user to the application you created. We are going to automate these steps.
+
+Finally, navigate to [breadcrumb]#Customizations -> Themes# and preview the FusionAuth theme. The application you created in the 5-minute guide uses the OAuth authorize and OAuth logout pages.
+
+The OAuth authorize page looks like this:
+
+image::themes/kickstart-theme/preview-oauth.png[Preview theme,width=1200]
```

Comment:
This has an exception. Can you please fix?

Reply:
I don't see it on the sandbox.fusionauth.io. Can you try upgrading to 1.42.0 and see if the issue still occurs? I know there were some webauthn changes that happened between 1.41.0 and 1.42.0.